util/types/time.go

```diff
@@ -35,6 +37,11 @@ var (
 	ErrInvalidYear       = errors.New("invalid year")
 )
 
+// ErrIncorrectDatetimeValue generate an 'Incorrect datetime value: XXX' error
+func ErrIncorrectDatetimeValue(v string) error {
```

Comment:
you can use `types.ErrIncorrectDatetimeValue` defined in file "util/types/time.go"